packages/nextjs/src/config/index.ts

```diff
@@ -96,6 +97,12 @@ class Config {
     return this.fronteggAppOptions.hostedLoginBox ?? false;
   }
 
+  get disableInitialPropsRefreshToken(): boolean {
+    const disableInitialPropsRefreshToken =
+      getEnv(EnvVariables.DISABLE_INITIAL_PROPS_REFRESH_TOKEN) ?? setupEnvVariables.DISABLE_INITIAL_PROPS_REFRESH_TOKEN;
```

Comment:
Both getEnv and setupEnvVariables are `process.env.DISABLE_INITIAL_PROPS_REFRESH_TOKEN` so no need for the fallback

Reply:
no, one for build time and other for runtime. 
it was fallback for edge runtime